Index: VTK-9.0.3/Common/Core/vtkGenericDataArrayLookupHelper.h
===================================================================
--- VTK-9.0.3.orig/Common/Core/vtkGenericDataArrayLookupHelper.h
+++ VTK-9.0.3/Common/Core/vtkGenericDataArrayLookupHelper.h
@@ -24,6 +24,7 @@
 
 #include "vtkIdList.h"
 #include <algorithm>
+#include <limits>
 #include <cmath>
 #include <unordered_map>
 #include <vector>
Index: VTK-9.0.3/Common/DataModel/vtkPiecewiseFunction.cxx
===================================================================
--- VTK-9.0.3.orig/Common/DataModel/vtkPiecewiseFunction.cxx
+++ VTK-9.0.3/Common/DataModel/vtkPiecewiseFunction.cxx
@@ -24,6 +24,7 @@
 #include <iterator>
 #include <set>
 #include <vector>
+#include <limits>
 
 vtkStandardNewMacro(vtkPiecewiseFunction);
 
Index: VTK-9.0.3/Rendering/Core/vtkColorTransferFunction.cxx
===================================================================
--- VTK-9.0.3.orig/Rendering/Core/vtkColorTransferFunction.cxx
+++ VTK-9.0.3/Rendering/Core/vtkColorTransferFunction.cxx
@@ -23,6 +23,7 @@
 #include <iterator>
 #include <set>
 #include <vector>
+#include <limits>
 
 vtkStandardNewMacro(vtkColorTransferFunction);
 
Index: VTK-9.0.3/Filters/HyperTree/vtkHyperTreeGridThreshold.cxx
===================================================================
--- VTK-9.0.3.orig/Filters/HyperTree/vtkHyperTreeGridThreshold.cxx
+++ VTK-9.0.3/Filters/HyperTree/vtkHyperTreeGridThreshold.cxx
@@ -28,6 +28,7 @@
 #include "vtkHyperTreeGridNonOrientedCursor.h"
 
 #include <cmath>
+#include <limits>
 
 vtkStandardNewMacro(vtkHyperTreeGridThreshold);
 
