diff -urN a/CMake/FindTBB.cmake b/CMake/FindTBB.cmake
--- a/CMake/FindTBB.cmake	2021-07-02 18:31:21.000000000 +0300
+++ b/CMake/FindTBB.cmake	2022-02-15 11:33:04.773834215 +0300
@@ -401,7 +401,7 @@
 
  #only read the start of the file
  file(STRINGS
-      "${TBB_INCLUDE_DIR}/tbb/tbb_stddef.h"
+      "${TBB_INCLUDE_DIR}/tbb/version.h"
       TBB_VERSION_CONTENTS
       REGEX "VERSION")
 
diff -urN a/ThirdParty/vtkm/vtkvtkm/vtk-m/CMake/FindTBB.cmake b/ThirdParty/vtkm/vtkvtkm/vtk-m/CMake/FindTBB.cmake
--- a/ThirdParty/vtkm/vtkvtkm/vtk-m/CMake/FindTBB.cmake	2021-07-02 18:31:21.000000000 +0300
+++ b/ThirdParty/vtkm/vtkvtkm/vtk-m/CMake/FindTBB.cmake	2022-02-15 11:37:46.788502241 +0300
@@ -337,7 +337,7 @@
 
  #only read the start of the file
  file(READ
-      "${TBB_INCLUDE_DIR}/tbb/tbb_stddef.h"
+      "${TBB_INCLUDE_DIR}/tbb/version.h"
       TBB_VERSION_CONTENTS
       LIMIT 2048)
 
diff -urN a/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/cont/tbb/internal/FunctorsTBB.h b/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/cont/tbb/internal/FunctorsTBB.h
--- a/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/cont/tbb/internal/FunctorsTBB.h	2021-07-02 18:31:21.000000000 +0300
+++ b/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/cont/tbb/internal/FunctorsTBB.h	2022-02-15 11:37:28.883845305 +0300
@@ -38,7 +38,7 @@
 // correct settings so that we don't clobber any existing function
 #include <vtkm/internal/Windows.h>
 
-#include <tbb/tbb_stddef.h>
+#include <tbb/version.h>
 #if (TBB_VERSION_MAJOR == 4) && (TBB_VERSION_MINOR == 2)
 //we provide an patched implementation of tbb parallel_sort
 //that fixes ADL for std::swap. This patch has been submitted to Intel
